docs/guide/sending-transactions.md

```diff
@@ -56,10 +56,11 @@ sendEthButton.addEventListener('click', () => {
       params: [
         {
           from: accounts[0],
-          to: '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
-          value: '0x29a2241af62c0000',
-          gasPrice: '0x09184e72a000',
-          gas: '0x2710',
+          to: '0x0c54FcCd2e384b4BB6f2E405Bf5Cbc15a017AaFb',
+          value: '0x0',
+          gasLimit: '0x5028',
+          gasPrice: '0x2540be400',
+          type: '0x0',
```

Comment:
While we're making this change, should we take the opportunity to send an EIP-1559 transaction here instead of a type-0 transaction?